client/consensus/grandpa/authorities_test.go

```diff
@@ -1205,36 +1205,30 @@ func TestCleanUpStaleForcedChangesWhenApplyingStandardChangeAlternateCase(t *tes
 
 func assertExpectedSet(t *testing.T, authSetID authoritySetChangeID, expected setIDNumber[uint]) {
 	t.Helper()
-	authSetVal, err := authSetID.Value()
-	require.NoError(t, err)
-	switch val := authSetVal.(type) {
-	case set[uint]:
+	switch val := authSetID.(type) {
+	case authoritySetChangeIDSet[uint]:
 		require.Equal(t, expected, val.inner)
 	default:
-		err = fmt.Errorf("invalid authSetID type")
+		err := fmt.Errorf("invalid authSetID type")
+		require.NoError(t, err)
```

Comment:
nice!

Reply:
This is kinda a funny one, but open to suggestions if there is a better way to do this!